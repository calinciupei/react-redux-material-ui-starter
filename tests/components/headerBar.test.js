import React from 'react';
import { shallow } from 'enzyme';

import HeaderBar from '../../src/components/headerBar/headerBar.component';

const createComponent = () => shallow(<HeaderBar />);

describe('<HeaderBar />', () => {
  const context = createComponent();

  it('should render correctly', () => {
    expect(context).toMatchSnapshot();
  });
});
