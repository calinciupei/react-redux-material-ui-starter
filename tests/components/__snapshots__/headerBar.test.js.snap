// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderBar /> should render correctly 1`] = `
<div>
  Header Bar
</div>
`;
